site/model.md

```diff
@@ -0,0 +1,346 @@
+
+The model structure, and its folder and attribute names, are based on the WLST 12.2.1.3 offline structure and names with redundant folders removed to keep the model simple.  For example, the WLST path to the URL for a JDBC data source is `/JDBCSystemResource/<data-source-name>/JdbcResource/<data-source-name>/JDBCDriverParams/NO_NAME_0/URL`.  In the model, it is `resources:/JDBCSystemResource/<data-source-name>/JdbcResource/JDBCDriverParams/URL` (where `resources` is the top-level model section where all WebLogic Server resources/services are described).
+
+The model is written in YAML (or optionally, JSON).  The YAML parser, built into the underlying framework, is both strict with regard to the specification and supports only the subset of YAML needed to describe WebLogic Server artifacts.  For example, YAML does not support tabs as indent characters so the parser will generate parse errors if the model file contains leading tabs used for indention purposes.  In general, names and values can be specified without quotes except when the content contains one of the restricted characters; in which case, the content must be enclosed in either single or double quotes.  The restricted characters are:
+
+- comma
+- colon
+- exclamation mark
+- question mark
+- hyphen
+- ampersand
+- percent sign
+- "at" sign
+- star
+- pound sign (also known as hash)
+- equal sign
+- less than
+- greater than
+- square brackets
+- curly braces
+- back quote
+
+All assignment statements must have one or more spaces between the colon and the value.  All comments must have a space after the pound sign (also known as hash) to be considered a comment.  YAML doesn't allow comments in all locations.  While the YAML parser used by the framework does not try to enforce these restrictions, it is likely that putting comments in some locations may cause parse errors since YAML is a difficult language to parse due to its complex indention rules.
+
+## Contents
+
+- [Top-Level Sections](#top-level-model-sections)
+- [Simple Example](#simple-example)
+- [Model Names](#model-names)
+- [Model Semantics](#model-semantics)
+- [Declaring Named MBeans to Delete](#declaring-named-mbeans-to-delete)
+- [Using Multiple Models](#using-multiple-models)
+- [The Archive File](site/archive.md)
```

Comment:
Breaking the model section into a separate file really improves the top-level page a lot.
"The Archive File" link seems a little buried here now. Anybody else think it should be linked to the top page instead, maybe under "Concepts" on the top page?

Reply:
I agree with you, it is hard to find stuck under models.

Reply:
I agree. If the Archive File information needs greater visibility, then we should move its cross-reference to the Home page under Concepts.